src/styles/main.css

```diff
@@ -10,9 +10,18 @@ html {
 }
 
 * {
-  -webkit-transition: border 200ms linear, background-color 200ms linear;
-  -moz-transition: border 200ms linear, background-color 200ms linear;
-  -o-transition: border 200ms linear, background-color 200ms linear;
-  -ms-transition: border 200ms linear, background-color 200ms linear;
```

Comment:
the claim is that this didn't pass lint before? any thoughts how lint didn't prevent it earlier? maybe that we added the linter after this was written and no one ever changed this file since then. if it's that, we should take some time later to run it on the whole codebase.

Reply:
This change was actually committed by the megalinter service automatically. I assume a new version of megalinter was released, we don't pin the megalinter version and this PR was the first one running it.